shared/oval/sshd_use_approved_ciphers.xml

```diff
@@ -4,12 +4,12 @@
       <title>Use Only Approved Ciphers</title>
       <affected family="unix">
         <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_debian</platform>
       </affected>
       <description>Limit the ciphers to those which are FIPS-approved and only
       use ciphers in counter (CTR) mode.</description>
     </metadata>
     <criteria operator="AND">
-      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
```

Comment:
this would prevent this check from working on non-certified systems, there is already a separate xccdf rule (installed_OS_is_certified) to determine whether the OS is certified or not, although it does not seem to be in use.

both, RHEL 7 ([v 2.1.0](https://benchmarks.cisecurity.org/tools2/linux/CIS_Red_Hat_Enterprise_Linux_7_Benchmark_v2.1.0.pdf)) and Debian 8 ([v 1.0.0](https://benchmarks.cisecurity.org/tools2/linux/CIS_Debian_Linux_8_Benchmark_v1.0.0.pdf)) CIS benchmarks, contain this recommendation:
_RHEL 7_ (**5.2.11** Ensure only approved ciphers are used):
`Ciphers aes256-ctr,aes192-ctr,aes128-ctr,aes256-gcm@openssh.com,aes128-
gcm@openssh.com,chacha20-poly1305@openssh.com`
_Debian 8_: (**9.3.11** Use Only Approved Cipher in Counter Mode)
`Ciphers aes128-ctr,aes192-ctr,aes256-ctr`

I'd like to either:
- remove the criterion, and add the XCCDF `installed_OS_is_certified` to profiles where `sshd_use_approved_ciphers` or `sshd_use_approved_macs` is used, or
- rename this (and `sshd_use_approved_macs`) OVAL/XCCDF to `sshd_use_approved_ciphers_fips` and add a new `sshd_use_approved_ciphers` check without the `installed_OS_is_certified` criterion, which can be used for Debian,etc

Which one would you prefer?

I'd also like to add to add the aes-gcm ciphers and/or chacha20-poly1305 ciphers.